lib/command.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2020 Balena Ltd.
+ * Copyright 2016-2022 Balena Ltd.
```

Comment:
I've set the copyright notices to be the same across the board, since there doesn't seem to be any value in tracking individual start dates.

Reply:
Right, just using the first date sounds sensible.